src/components/common/organisms/ServiceMessage.tsx

```diff
@@ -11,55 +11,91 @@ const ServiceMessage = (props: {
   const [serviceMessage, setServiceMessage] = useState<
     ServiceMessage | 'REQUEST FAILED'
   >(null);
+  const [serviceMessageShown, setServiceMessageShown] = useState<boolean>(true);
   const safeAreaInsets = useSafeAreaInsets();
 
+  const lastInterval = useRef<NodeJS.Timer>();
+
+  useEffect(() => {
+    function fetchServiceMessage(environment: string) {
+      fetch(
+        `https://api.statoil.com/app/mad/${environment}api/v1/ServiceMessage/${props.serviceName}`
+      )
+        .then((res) =>
+          res.json().then((data: ServiceMessage) => {
+            if (
+              !serviceMessage ||
+              (serviceMessage !== 'REQUEST FAILED' &&
+                data.alertName !== serviceMessage.alertName)
+            ) {
+              setServiceMessage(data);
+              setServiceMessageShown(true);
+            }
+          })
+        )
+        .catch(() => setServiceMessage('REQUEST FAILED'));
+    }
+
+    const environment: string =
+      props.environment.toLowerCase() === 'prod' ? `` : `${props.environment}/`;
+
+    if (!lastInterval.current) fetchServiceMessage(environment);
+    if (lastInterval.current) clearInterval(lastInterval.current);
+    lastInterval.current = setInterval(() => {
+      console.log('SERVICE MESSAGE:', serviceMessage);
+      fetchServiceMessage(environment);
+    }, 300000);
+
+    return () => {
+      clearInterval(lastInterval.current);
+    };
+  }, [serviceMessage]);
+
   const displayBanner = () => {
     if (serviceMessage === 'REQUEST FAILED')
       return (
         <>
+          <View>
+            <Banner
+              viewStyle={StyleSheet.flatten([
+                styles.bannerViewStyle,
+                {
+                  paddingTop: safeAreaInsets.top,
+                  minHeight: 80 + safeAreaInsets.top,
+                },
+              ])}
+              maxExpandedHeight={400}
+              maxNonExpandedHeight={80 + safeAreaInsets.top}
+              text={'Could not retrieve service message'}
```

Comment:
Should this be in a language dict for our apps with multiple languages?

Reply:
Yes, should be resolved in the 'f/norwegianLanguage' branch.

Reply:
@HavardNot Is it resolved inside the service message component in another branch? Otherwise we might as well add it here

Reply:
No, you right. The lang support for servicemessage can be resolved here, but we also need to make changes to E&P to pass language code to the component. That can be implemented in the f/norwegianlanguage branch of E&P